kobo/apps/accounts/mfa/tests/test_api.py

```diff
@@ -67,3 +71,21 @@ def test_mfa_activation_always_creates_new_secret(self):
             )
             assert first_secret != second_secret
             assert first_response.json() != second_response.json()
+        mfa_availability.delete()
+
+    def test_mfa_disabled(self):
+        anotheruser = User.objects.get(username='anotheruser')
+        self.client.login(username='anotheruser', password='anotheruser')
+        method = list(trench_settings.MFA_METHODS.keys())[0]
+
+        activate_response = self.client.post(
+            reverse('mfa-activate', args=(method,))
+        )
+        assert activate_response.status_code == 403
```

Comment:
Let's use DRF constants instead, i.e. `status.HTTP_403_FORBIDDEN'. 
Please, change others accordingly.

Reply:
BTW, I'm wondering if the permission class should return a 400 instead of 403? I think it sounds more like a bad request than a forbidden access? 
What do you think?

Reply:
I think it's both. To me it looks like if you are not in the MFA available  to user table, then you don't have the permission to use MFA. I agree that 400 makes sense too, but I'm more inclined to the 403, as in: you are not allowed to activate MFA.

Reply:
So let's keep the 403, then.